### ISSUE TITLE
Can this extension be marked parallel read and/or write safe?

### ISSUE THREAD

**Issue by jdillard** [bavovanachte/sphinx-wavedrom#39]
I get the following warning:

```
WARNING: the sphinxcontrib.wavedrom extension does not declare if it is safe for parallel reading, assuming it isn't - please ask the extension author to check and make it explicit
```

Adding `parallel_read_safe` (and `parallel_write_safe` if that is also true) to the [setup](https://github.com/bavovanachte/sphinx-wavedrom/blob/master/sphinxcontrib/wavedrom.py#L185) definition will fix that warning:

```
return {
    'parallel_read_safe': True,
    'parallel_write_safe': True
}
```
If that is the case, I don't mind making a PR to add them.

**Comment by bavovanachte** [comment]
Honestly, I have no way to be sure. 

- I've checked the plantuml plugin, which this plugin took a lot of initial inspiration from, and that seems to be parallel-read-safe.
- The plugin doesn't seem to handle any of the core sphinx events in the parallel stages (5-8) (https://www.sphinx-doc.org/en/master/extdev/appapi.html?highlight=env_before_read_docs#sphinx-core-events), but I also read that that doesn't necessarily guarantee that it's safe (https://github.com/sphinx-doc/sphinx/issues/8829)

**Comment by jdillard** [comment]
I'm fairly positive this extension is parallel read/write safe as well. It doesn't seem to do any "cross page" calculations.

**Comment by bavovanachte** [comment]
Fixed in #40. Thanks again @jdillard !

**Comment by jdillard** [comment]
No problem! Thanks for being a maintainer!
